packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -303,7 +306,7 @@ export async function editSDKConnection(
     "includeDraftExperiments",
     "includeExperimentNames",
     "includeRedirectExperiments",
-    "remoteEvalEnabled",
```

Comment:
why is this removed? even if the underlying payload format isn't changing, we still signal a cache eviction / webhook to things like proxy, CDN, etc

Reply:
There were two instances of `remoteEvalEnabled` in the list so I got rid of one of them when adding the new feature

Reply:
nvm: duplicate field